library/kapacitor

```diff
@@ -1,7 +1,7 @@
 Maintainers: Jonathan A. Sternberg <jonathan@influxdata.com> (@jsternberg),
              j. Emrys Landivar <landivar@gmail.com> (@docmerlin)
 GitRepo: git://github.com/influxdata/influxdata-docker
-GitCommit: 63fe0684150d96916514a84cf5978c3512dc15fa
+GitCommit: 0e03b615c48d7f089fff7bb2aa2e7d65e83bd2c2
 
 Tags: 1.4, 1.4.1
```

Comment:
Just want to double check that 1.4.x is still actively supported (and not EOL now that 1.6.x is released).

As a reminder, removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)